.github/ISSUE_TEMPLATE/iceberg_proposal.yml

```diff
@@ -0,0 +1,50 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: Iceberg Improvement Proposal
+description: Propose a Spec change or major feature
+labels: ["proposal"]
+body:
+  - type: markdown
+    attributes:
+      value: "Please see documentation site for information on [contributing proposals](https://iceberg.apache.org/contribute/#iceberg-improvement-proposals)"
+  - type: textarea
+    attributes:
+      label: Proposed Change
+      description: Please describe the proposal and elaborate on the use case and motivation
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Proposal document
+      description: |
+        Link to the proposal document.  Google Docs is preferred format to allow for public 
+        comment and sharing
+  - type: checkboxes
+    attributes:
+      label: Specifications
+      description: Which specifications are affected by this proposal?
+      options:
+        - label: Table
+        - label: View
+        - label: REST
+        - label: Puffin
+        - label: Encryption
```

Comment:
Should be this `Security`?

Reply:
I think this specifically refers to the Encryption spec: https://iceberg.apache.org/gcm-stream-spec/